workflows/diagnostics/fv3net/diagnostics/offline/_helpers.py

```diff
@@ -143,11 +150,32 @@ def insert_rmse(ds: xr.Dataset):
 
 
 def load_grid_info(res: str = "c48"):
+    if "ne" in res:
+        return load_grid_info_scream(res)
+    elif "c" in res:
```

Comment:
We should probably define this convention somewhere centralized (in `vcm`?) instead of having similar functions that distinguish between scream and fv3 "res" strings in many places.

I'd also suggest checking `res.startswith("ne")` and `res.startswith("c")` instead of `in`. (Maybe another grid will come along with a `c` in it somewhere!)